### PR TITLE
Update freelancers.json

### PR DESCRIPTION
**Pull request by campossrg**
Hello, I would like to add my info to the https://bcneng.org/freelancers/ stack. Thank you so much :D

### CODE REVIEW

**Review by sdecandelario — APPROVED** [review]

File: data/freelancers.json [hunk 1]
```diff
@@ -75,5 +75,16 @@
...
         "availability_hours_week": {"min": 2, "max": 10},
         "pricing_model": "Depending on the service requested"
+    },
+        {
+        "slack_name": "@Sergio Campos",
+        "slack_member_id": "U01U6401XMX",
+        "full_name": "Sergio Campos",
+        "web": "campossrg.com",
+        "linkedin_handler": "campossrg",
+        "services": "Java backend developer. Web developer. ",
```
Comment (smoya) [inline data/freelancers.json:85]:
```suggestion
        "services": "Java backend developer. Web developer.",
```

File: data/freelancers.json [hunk 2]
```diff
@@ -75,5 +75,16 @@
...
+        "skills": ["Java, Spring framework, Oracle, SQL Server, Javascript, CI/CD, Docker"],
```
Comment (smoya) [inline data/freelancers.json:86]: This is an array so each array element should be a string delimited by quotes (or double). You just created an array with one element here.

**Review by smoya — CHANGES_REQUESTED** [review]

File: data/freelancers.json (hunk 2 above)
Comment (campossrg) [reply data/freelancers.json]: Thank you for the correction ;)

**Review by campossrg — COMMENTED** [review]
Update last changes

**Pull request merged**